src/Rules/Resources/ConditionDescriptions.resx

```diff
@@ -245,4 +245,7 @@
   <data name="IsDialog" xml:space="preserve">
     <value>IsDialog</value>
   </data>
+  <data name="LocalizedLandmarkType" xml:space="preserve">
+    <value>LocalizedLandmarkType</value>
+  </data>
 </root>
```

Comment:
Should this file have a final newline?

Reply:
It was edited via the IDE, so I'm assuming that the file formatting is normalized